### PR TITLE
Ui updates

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
-Filters out unverified breaches that are classified as spam, retired, and/or sensitive. 
-Disables input while email scan form is processing.

### CODE REVIEW

File: public/js/monitor.js [hunk 1]
```diff
@@ -158,6 +158,9 @@ async function hashEmailAndSend(emailFormSubmitEvent) {
   for (const emailInput of emailForm.querySelectorAll("input[type=email]")) {
     emailForm.querySelector("input[name=emailHash]").value = await sha1(emailInput.value);
     emailInput.value = "";
+    emailInput.disabled = true;
+    emailInput.placeholder="";
```
Comment (pdehaan) [inline public/js/monitor.js]: minorest of nits, but spaces around the `=` would make it easier to read (and more consistent).

File: hibp.js [hunk 2]
```diff
@@ -37,7 +37,12 @@ const HIBP = {
     } catch (error) {
       console.error(error);
     }
-    return foundBreaches;
+    for (const breach of foundBreaches) {
+      if ( breach.meta.IsVerified && !breach.meta.IsRetired && !breach.meta.IsSensitive && !breach.meta.IsSpamList ) {
+        filteredBreaches.push(breach);
+      }
+    }
+    return filteredBreaches;
```
Comment (pdehaan) [inline hibp.js]: Silly comment, but not sure if it's easier to create a second array versus just doing something like this pseudo-shenanigans:

```js
return foundBreaches.filter(({meta}) => meta.IsVerified && !meta.IsRetired && !meta.IsSensitive && !meta.IsSpamList);
```

Comment (nhnt11) [reply hibp.js]: Why silly? I was going to suggest this :)

File: hibp.js [hunk 3]
```diff
@@ -37,7 +37,12 @@ const HIBP = {
     } catch (error) {
       console.error(error);
     }
-    return foundBreaches;
+    for (const breach of foundBreaches) {
+      if ( breach.meta.IsVerified && !breach.meta.IsRetired && !breach.meta.IsSensitive && !breach.meta.IsSpamList ) {
```
[the remark this answers is not in the document]
Comment (nhnt11) [reply hibp.js]: Great insight, thanks! I've reached out to Troy for his comment on this.

File: public/js/monitor.js (hunk 1 above)
Comment (lesleyjanenorton) [reply public/js/monitor.js]: hahaha i'll fix it. 'minorest of nits' might be my new favorite phrase.

File: hibp.js (hunk 2 above)
Comment (lesleyjanenorton) [reply hibp.js]: way better! not silly at all.

**Comment by pdehaan** [comment]
@nhnt11 Did we get any response from Troy as to what results we should show (re: `.IsVerified` and `.IsSpamList`)?

**Comment by nhnt11** [comment]
Update: I proposed that we exclude breaches that are (spam lists || unverified || sensitive || retired); Troy agreed.

**Comment by groovecoder** [comment]
Looks good. I updated some of my breaches to be spam, unverified, sensitive, and/or retired and the filter worked correctly on all cases.

**Comment by lesleyjanenorton** [comment]
I'm still confused but I think the decision to filter out unverified breaches was partly in response to user testing videos where users were totally confused by the unverified breaches. In a few cases, users remarked that they'd never had an account on 'Anti Public Combo List' (or whatever the unverified breach was) and then started questioning all of the other results. If a decision is made to list and flag them later we'll probably need to do something similar to HIBP and provide additional information about what 'unverified' means in this context as well.

**Pull request merged**